addons/website/models/ir_module_module.py

```diff
@@ -152,7 +152,9 @@ def _update_records(self, model_name, website):
                 # special case for attachment
                 # if module B override attachment from dependence A, we update it
                 if not find and model_name == 'ir.attachment':
-                    find = rec.copy_ids.search([('key', '=', rec.key), ('website_id', '=', website.id)])
+                    # In master, a unique constraint over (theme_template_id, website_id)
+                    # will be introduced, thus ensuring unicity of 'find'
+                    find = rec.copy_ids.search([('key', '=', rec.key), ('website_id', '=', website.id), ("original_id", "=", False)], order='create_date ASC', limit=1)
```

Comment:
Now that we figured the issue and know it comes from the fact that before [this filx](https://github.com/odoo/odoo/commit/254d40ffcc21f56a80a1814da7f6355d44e5670b), some records could be copied with their `theme_template_id` (making it also copies) through the website editor, especially with image manipulation like crop. Why don't you just add the real fix which is `("original_id", "=", False)` (like @KangOl suggested back then)?

The order and the limit are useless, you will only get one result now that you exclude the "broken" record(s) by inserting `("original_id", "=", False)`, isn't it?